### DISCUSSION TITLE
Eagler Reborn Status Update

### DISCUSSION THREAD

**Discussion by leahanderson1** [EaglerReborn/support#13] [Announcements]
# Eagler Reborn Status Update

It's been a while since I've really given any updates, so I thought I would make one.

## Eagler Reborn 1.0

The first non-beta release for Eagler Reborn. I expect it to come out around mid-December to early 2024. I will reveal more details on it later.

## The Web Porting Project
![image](https://github.com/EaglerReborn/support/assets/144633084/9cd27073-8ead-4046-995a-a309baf5050d)

This project is for porting games to WebAssembly. This is what I'll mainly be working on from here on out. I'm planning to port 90's to early-2000's games. If you have any suggestions, let me know! You can find a link to the organization [here.](https://github.com/Web-Porting-Project)

**Comment by wxnnvs** [comment]
Will the pluginapi be updated too?
And what features are gonna be in v1

**Reply by leahanderson1** [reply]
I'm fixing the bug [here,](https://github.com/radmanplays/MeteorX-ts/issues/6) for one.

**Reply by radmanplays** [reply]
i found another bug that breaks the autoclicker module in my hacked client : 
![image](https://github.com/EaglerReborn/support/assets/95340057/1a272625-c4dc-479d-8148-3570a7e197b4)

**Reply by ghost** [reply]
What do you mean? Is it like a Client for Hacking? Like to hack in Bedwars Lol Sorry I am just Curious.

**Reply by wxnnvs** [reply]
> What do you mean? Is it like a Client for Hacking? Like to hack in Bedwars Lol Sorry I am just Curious.

It is a client allowing for custom client side plugins, kind of like a modloader instead of hardcoded mods in the source

**Reply by ghost** [reply]
Oh Okay thx

**Comment by wxnnvs** [comment]
How's it going?

**Reply by ghost** [reply]
It is going Pretty good, the Client it is Working, But I rerighted the Code of BSg=Enchantment.java:177:1, I just rewrote the Code, Lol

**Reply by leahanderson1** [reply]
1.0 will also feature a sort of a5.3.5 made by me as well, porting the APIs i made back in indev.

**Reply by ghost** [reply]
Oh Nice,
